src/transport/raw/MessageHeader.h

```diff
@@ -49,6 +49,22 @@ enum class EncryptionType
     kAESCCMTagLen16 = 2,
 };
 
+/**
+ *  @brief
+ *    The CHIP Exchange header flag bits.
+ */
+enum class ExFlagValues : uint16_t
+{
+    /// Set when current message is sent by the initiator of an exchange.
+    kExchangeFlag_Initiator = 0x0001,
+
+    /// Set when current message is an acknowledgment for a previously received message.
+    kExchangeFlag_AckId = 0x0002,
+
+    /// Set when current message is requesting an acknowledgment from the recipient.
+    kExchangeFlag_NeedsAck = 0x0004,
```

Comment:
why is this flag part of exchange manager?

Reply:
Same as above.